src/main.cpp

```diff
@@ -0,0 +1,206 @@
+#include <vector>
+#include <iostream>
+#include <string>
+#include <windows.h>
+
+#include "pugixml.hpp"
+
+#define SDL_MAIN_HANDLED
+#include <SDL.h>
+
+const int screen_width = GetSystemMetrics(SM_CXSCREEN);
+const int screen_height = GetSystemMetrics(SM_CYSCREEN);
+
+enum StatusCode {
+    SUCCESS = 0,
+    FILENAME_NOT_PROVIDED,
+    PARSER_FAILURE,
+    SDL_INIT_FAILURE
+};
+
+const char *status_messages[] = {
+        "Success\n",
+        "Filename was not provided\n",
+        "Parser failure\n",
+        "SDL could not be initialized\n"
+};
+
+const char *parser_e_id = "e_id";
+const char *parser_c_id = "c_id";
+const char *parser_x = "x";
+const char *parser_y = "y";
+const char *parser_height = "height";
+const char *parser_width = "width";
+const char *parser_end_elem = "end_element";
```

Comment:
`const char *` are mutable pointers to immutable data, you have better options for the task:
- `const char * const varname = ...`
- `const char varname[] = ...`
- `const std::string varname(...)`